eth/catalyst/api.go

```diff
@@ -163,15 +197,56 @@ func (api *ConsensusAPI) ForkchoiceUpdatedV1(update beacon.ForkchoiceStateV1, pa
 		data, err := api.assembleBlock(update.HeadBlockHash, payloadAttributes)
 		if err != nil {
 			log.Error("Failed to create sealing payload", "err", err)
-			return beacon.ForkChoiceResponse{Status: beacon.VALID}, err // Valid as setHead was accepted
+			return beacon.STATUS_INVALID, err // Valid as setHead was accepted
 		}
 		id := computePayloadId(update.HeadBlockHash, payloadAttributes)
 		api.localBlocks.put(id, data)
 
 		log.Info("Created payload for sealing", "id", id, "elapsed", time.Since(start))
-		return beacon.ForkChoiceResponse{Status: beacon.VALID, PayloadID: &id}, nil
+		return api.validFCU(&id), nil
```

Comment:
Please don't use weird abbreviations. LES is already impossible to read without a cheat sheet.